packages/blade/src/components/Spinner/BaseSpinner/BaseSpinner.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { dimensions } from './spinnerTokens';
+import SpinnerIcon from './SpinnerIcon';
+import { SpinningBox } from './SpinningBox';
+import type { Theme } from '~components/BladeProvider';
+import { useTheme } from '~components/BladeProvider';
+import { getIn, makeAccessible, makeSize } from '~utils';
+import type { ColorContrastTypes, Feedback } from '~tokens/theme/theme';
+
+type BaseSpinnerProps = {
+  intent?: Feedback;
+  /**
+   * Sets the contrast of the spinner.
+   *
+   * @default 'low'
+   */
+  contrast?: ColorContrastTypes;
+  /**
+   * Sets the size of the spinner.
+   *
+   * @default 'medium'
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Sets the aria-label for web & accessibilityLabel react-native.
+   *
+   */
+  accessibilityLabel: string;
+  /**
+   * Indicates that an element will be updated, and describes the types of updates the user agents, assistive technologies, and user can expect from the live region.
+   *
+   *  @default 'assertive'
+   */
+  accessibilityLiveRegion?: 'off' | 'assertive' | 'polite';
```

Comment:
Let's get derive from the makeAccessible types: 

accessibilityLiveRegion: AccessibilityProps['liveRegion']